utoipa-swagger-ui/src/lib.rs

```diff
@@ -758,6 +766,24 @@ impl<'a> Config<'a> {
         self
     }
 
+    /// Set 'layout' to 'BaseLayout' to only use the base swagger layout without a search header.
+    ///
+    /// Default value is 'StandaloneLayout'.
+    ///
+    /// # Examples
+    ///
+    /// Allow operation id to be shown.
```

Comment:
I guess this is a copy paste after effect. Maybe "Change Swagger UI to BaseLayout"?

Reply:
True that 😅